client/extensions/woocommerce/app/products/product-header.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import StickyPanel from 'components/sticky-panel';
+
+const ProductHeader = ( { onTrash, onDuplicate, onSave, translate } ) => {
+	const trashButton = onTrash &&
+		<Button borderless onClick={ onTrash }><Gridicon icon="trash" /></Button>;
+
+	const duplicateButton = onDuplicate &&
+		<Button onClick={ onDuplicate }>{ translate( 'Duplicate' ) }</Button>;
+
+	const saveButton = onSave &&
+		<Button primary onClick={ onSave }>{ translate( 'Save' ) }</Button>;
+
+	return (
+		<StickyPanel>
+			<Card className="products__header">
+				<span>Breadcrumbs > go > here</span>
+				<div className="products__header-right">
```

Comment:
This is actually in the [HTML style guidelines](https://github.com/Automattic/wp-calypso/blob/de700a238a201232c8481b59b4f7feba7700e977/docs/coding-guidelines/html.md#craftsmanship) as well.

> The markup should not have style preferences, for the most part, so that changing how the CSS renders doesn't leave things with weird meanings (references to position or visuals, left, right, dark, blue).